scripts/download-helper.js

```diff
@@ -0,0 +1,101 @@
+const fs = require('fs');
+const https = require('https');
+const zlib = require('zlib');
+const path = require('path');
+const tar = require('tar');
+const sha384 = require('sha384');
+
+// TODO: Change URL from fork to @runk's repo (and master branch)
+const link = file => `https://raw.githubusercontent.com/GitSquared/node-geolite2/new-eula-redistribution/redist/${file}`;
+
+const editions = [
+  { name: 'GeoLite2-ASN' },
+  { name: 'GeoLite2-City' },
+  { name: 'GeoLite2-Country' }
+];
```

Comment:
Perhaps i'd do `const editions = [ ... ].map((edition) => ...)`

Reply:
Indeed, that's much better. I changed the code to use Array.map() whenever possible.